manager_D3S.py

```diff
@@ -311,33 +312,37 @@ def data_log(self, file, spectra):
             with open(file, 'a') as f:
                 f.write('{0}, '.format(spectra))
                 self.vprint(2, 'Writing spectra to data log at {}'.format(file))
-    
+
     def calibration_log(self, file, spectra):
         """
-        Writes spectra to calubration-log.
+        Writes spectra to calibration-log.
         """
         if self.calibrationlogflag:
             with open(file, 'a') as f:
                 f.write('{0}, '.format(spectra))
                 self.vprint(2, 'Writing spectra to calubration log at {}'.format(file))
-                
+            self.c_timer += self.interval
+            if self.c_timer >= 600:
```

Comment:
Yes.
The argparse options are getting pretty messy because we've added so much. I'm thinking about possible restructuring, but for now just add another argparse option. Thanks.